pkg/envoy/lds/egress_test.go

```diff
@@ -222,6 +225,9 @@ func TestGetEgressFilterChainsForMatches(t *testing.T) {
 				cfg: mockConfigurator,
 			}
 			mockConfigurator.EXPECT().IsTracingEnabled().Return(false).AnyTimes()
+			mockConfigurator.EXPECT().GetInboundExternalAuthConfig().Return(configurator.ExternAuthConfig{
+				Enable: false,
+			}).AnyTimes()
```

Comment:
Let's avoid this by only making this function call for the inbound direction.

Reply:
I guess you mean outbound, and pair it with your previous suggestion to only call `GetInboundExternalAuthConfig` when is inbound.

Good idea. this will remove a lot of `mockConfigurator` test code.